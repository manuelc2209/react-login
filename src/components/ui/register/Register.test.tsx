import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { Register } from '.';

it('renders snapshot correctly', () => {
    const tree = renderer.create(<Register />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Register />, div);
    ReactDOM.unmountComponentAtNode(div);
});
