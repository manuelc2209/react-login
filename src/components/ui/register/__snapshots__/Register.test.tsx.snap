// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot correctly 1`] = `
<div
  className="sc-kDTinF ZqIuj"
>
  <button
    className="sc-gsDKAQ cAawDM sc-pVTFL"
    disabled={true}
    onClick={[Function]}
    onMouseDown={[Function]}
    onMouseUp={[Function]}
    size="small"
  >
    <span
      className="sc-bdvvtL jvqAQd"
      disabled={true}
    >
      Submit
    </span>
  </button>
</div>
`;
